crates/web/frontend/src/components/editor/editor-error-overlay.tsx

```diff
@@ -0,0 +1,29 @@
+import { X } from "lucide-react";
+import { Button } from "../ui/button";
+import styles from "./editor.module.css";
+
+interface Props {
+	visibleBackdrop: boolean;
+	visibleMessage: boolean;
+	errorMessage: string | undefined;
+	onClose: () => void;
+}
+
+export const EditorErrorOverlay = ({
```

Comment:
move this file to `editor/overlay/editor-error-overlay.tsx`? or to `editor/overlay/error.tsx`? In order to structure more the editor folder, it seems that it have too many files with different purposes